File: sources/manager/system_manager_py/system_manager_py/system_manager_node.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

class SystemManager(Node):
    def __init__(self,node_name="system_manager"):
        super().__init__(node_name)
        self.get_logger().info("节点已启动：%s!" % node_name)
        
def main(args=None):
    rclpy.init(args=args) # 初始化rclpy
    node = SystemManager()  # 新建一个节点
    rclpy.spin(node) # 保持节点运行，检测是否收到退出指令（Ctrl+C）
    rclpy.shutdown() # 关闭rclpy

File: sources/manager/system_manager/app/system_manager_node.cpp
#include <rclcpp/rclcpp.hpp>

#include "system_manager/system_manager.hpp"

int main(int argc, char **argv)
{
  rclcpp::init(argc, argv);
  auto node = std::make_shared<SystemManager>(); // 创建对应节点的共享指针对象
  rclcpp::spin(node);                            // 运行节点，并检测退出信号
  rclcpp::shutdown();
  return 0;
}
